docs/templates/preprocessing/image.md

```diff
@@ -59,7 +59,7 @@ Generate batches of tensor image data with real-time data augmentation. The data
     - __flow(X, y)__: Takes numpy data & label arrays, and generates batches of augmented/normalized data. Yields batches indefinitely, in an infinite loop.
         - __Arguments__:
             - __X__: data.
-            - __y__: labels.
+            - __y__: labels (Optional). Can be None in case of fitting in unlabelled data. 
```

Comment:
"If None, no labels will be generated and only batches of data will be yielded by the generator.